2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -20,6 +20,45 @@
 
 function createBookList(books) {
   // your code goes in here, return the ul element
+  const list = document.createElement('ul');
+  document.body.appendChild(list);
+
+  for (const book of books) {
+    const li = document.createElement('li');
+    list.appendChild(li).style.fontSize = '20px';
+
+    const paragraph = document.createElement('p');
+    paragraph.innerText = `${book.title} by "${book.author}"`;
+    li.appendChild(paragraph);
+
+    const img = document.createElement('img');
+    if (book.title === 'The Design of Everyday Things') {
```

Comment:
This might be better outside of this function. You can add to your books additional properties like this `image: { source: './assets/image.jpg', alt: 'the most human human'` and then just use these properties here but can also be reused everywhere else.